Symfony/CS/Tests/Fixer/Contrib/AlignConstantEqualsFixerTest.php

```diff
@@ -0,0 +1,243 @@
+<?php
+
+namespace Symfony\CS\Tests\Fixer\Contrib;
+
+use Symfony\CS\Tests\Fixer\AbstractFixerTestBase;
+
+/**
+ * @author Sullivan Senechal <soullivaneuh@gmail.com>
+ */
+class AlignConstantEqualsFixerTest extends AbstractFixerTestBase
+{
+    /**
+     * @param string $expected
+     * @param string $input
+     *
+     * @dataProvider provideFixCases
+     */
+    public function testFix($expected, $input = null)
+    {
+        $this->makeTest($expected, $input);
+    }
+
+    /**
+     * @return array
+     */
+    public function provideFixCases()
+    {
+        return array(
+            array(
+                '<?php
+                const FIRST_CONSTANT          = 1;
+                const SECOND_CONSTANT         = 2;
+                const THIRD_AND_LAST_CONSTANT = 3;
+
+                $myVar = 1;
+                $mySecondVar = 1;
+                ',
+            ),
+            array(
+                '<?php
+                const FIRST_CONSTANT          = 1;
+                const SECOND_CONSTANT         = 2;
+                const THIRD_AND_LAST_CONSTANT = 3;
+                ',
+                '<?php
+                const FIRST_CONSTANT = 1;
+                const SECOND_CONSTANT = 2;
+                const THIRD_AND_LAST_CONSTANT = 3;
+                ',
+            ),
+            array(
+                '<?php
+                const FIRST_CONSTANT          = 1;
+                const SECOND_CONSTANT         = 2;
+                const THIRD_AND_LAST_CONSTANT = 3;
+                ',
+                '<?php
+                const FIRST_CONSTANT            = 1;
+                const SECOND_CONSTANT           = 2;
+                const THIRD_AND_LAST_CONSTANT   = 3;
+                ',
+            ),
+            array(
+                '<?php
+                const FIRST_CONSTANT          = 1;
+                const SECOND_CONSTANT         = 2;
+                const THIRD_AND_LAST_CONSTANT = 3;
+                ',
+                '<?php
+                const FIRST_CONSTANT          = 1;
+                const SECOND_CONSTANT         = 2;
+                const THIRD_AND_LAST_CONSTANT   = 3;
```

Comment:
That is why I strongly suggest to work on tokens instead of text (https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/1268#issuecomment-113132868).

What you could is:
1. use tokens instead of text, perhaps rewriting `AbstractAlignFixer`
2. rewrite `AbstractAlignFixer` to fix it, still working on text
3. accept the fact that used alignment is bigger that it could be and change tests.